temporian/core/evaluator.py

```diff
@@ -68,14 +65,15 @@ def evaluate(
   schedule = get_operator_schedule(features_to_compute)
 
   # materialize input data. TODO: separate this logic
-  materialized_input_data = {
-      input_event: (
-          input_event_spec
-          if isinstance(input_event_spec, event)
-          else read_csv_fn(input_event_spec, input_event.sampling())
-      )
-      for input_event, input_event_spec in input_data.items()
-  }
+  materialized_input_data = {}
+  for input_event, input_event_spec in input_data.items():
+    if not isinstance(input_event_spec, event):
+      input_event_spec = read_csv_fn(input_event_spec, input_event.sampling())
+
+    materialized_input_data[input_event.sampling(
+    )] = input_event_spec.index  # TODO: handle samplings as different type of nodes instead of doing this
+    materialized_input_data[input_event] = input_event_spec
+
   # evaluate schedule
   outputs = evaluate_schedule_fn(materialized_input_data, schedule)
```

Comment:
Nit: What about having comments on a separate line (e.g., above) so as to limit the line length?